models/sushiswap/ethereum/sushiswap_ethereum_schema.yml

```diff
@@ -0,0 +1,91 @@
+version: 2
+
+models:
+  - name: sushiswap_ethereum_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: sushiswap
+      contributors: jeff-dude, masquot, soispoke, augustog
+    config:
+      tags: ['ethereum','sushiswap','trades','dex']
+    description: >
+        Uniswap V1 contract trades on Ethereum
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
+      - &project 
+        name: project
+        description: "Project name of the DEX"  
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the DEX project"
+      - &block_date
+        name: block_date
+        description: "UTC event block date of each DEX trade"
+      - &block_time
+        name: block_time
+        description: "UTC event block time of each DEX trade"
+      - &token_bought_symbol
+        name: token_bought_symbol
+        description: "Token symbol for token bought in the transaction"
+      - &token_sold_symbol
+        name: token_sold_symbol
+        description: "Token symbol for token sold in the transaction"
+      - &token_pair
+        name: token_pair
+        description: "Token symbol pair for each token involved in the transaction"
+      - &token_bought_amount
+        name: token_bought_amount
+        description: "Value of the token bought at time of execution in the original currency"
+      - &token_sold_amount
+        name: token_sold_amount
+        description: "Value of the token sold at time of execution in the original currency"
+      - &token_bought_amount_raw
+        name: token_bought_amount_raw
+        description: "Raw value of the token bought at time of execution in the original currency"
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
+        description: "Raw value of the token sold at time of execution in the original currency"
+      - &amount_usd
+        name: amount_usd
+        description: "USD value of the trade at time of execution"
+      - &token_bought_address
+        name: token_bought_address
+        description: "Contract address of the token bought"
+        tests:
```

Comment:
in order for this test to run, you'll need to add sushiswap tx's to our dex trades seed file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

there are uniswap examples in there. typically on the app UI, you can find an analytics page, maybe this one:
https://analytics.sushi.com/
which links to tx's ideally, where you can grab info and apply to seed file

Reply:
that's a tough one to find in etherscan (at least for me too). i believe it's under the ```logs``` tab -- you can use v1 engine to query that ```tx_hash```, find ```evt_index``` values and try to find on that tab in etherscan